src/CocktailDetails/CocktailDetails.tsx

```diff
@@ -76,16 +70,16 @@ const CocktailDetails: React.FC<CocktailDetailsProps> = ({ id, toggleFavorites,
         />
         <section className="info-wrapper">
           <p className="detail-paragraph">
-            <div className="sub-heading">Type: </div> {cocktailInfo.strCategory}
+            <span className="sub-heading">Type: </span> {cocktailInfo.strCategory}
             , {cocktailInfo.strAlcoholic}
           </p>
           <p className="detail-paragraph">
-            <div className="sub-heading">Glass: </div>
+            <span className="sub-heading">Glass: </span>
             {cocktailInfo.strGlass}
           </p>
           <h4>Instructions:</h4>
           <p>{cocktailInfo.strInstructions}</p>
-          <div className="sub-heading">Ingredients:</div>
+          <span className="sub-heading">Ingredients:</span>
```

Comment:
code was mad we had div's inside p tags, but a span is fine i suppose 🤷

Reply:
haven't used span's too much but glad it fixed the issue!

Reply:
This brings me back to Mod 1 for a project - span is completely invisible to screen readers, which is why we *shouldn't*  use it much.